object-construction-checker/tests/socket/HdfsReport1.java

```diff
@@ -0,0 +1,11 @@
+// Based on a false positive in hdfs
```

Comment:
Should we include this unrelated test in this PR? I would think it would go in a separate PR.